### ISSUE TITLE
Integration with Nextjs

### ISSUE THREAD

**Issue by gasPoweredSticks** [Tap-Payments/goSell-JSLib-Documentation#18]
Hey Tap team, 

I've tried to integrate your package with our Nextjs application however I've come across this issue by importing GoSell `import { GoSell } from "@tap-payments/gosell";`

<img width="475" alt="Screen Shot 2021-04-14 at 14 59 25" src="https://user-images.githubusercontent.com/69243854/114706734-09023f80-9d32-11eb-9467-2675e7c04984.png">

Does your package not work well with SSR? any pointers would be very much appreciated.

**Comment by mohamedhelmi70** [comment]
I have the same issue

**Comment by halaq** [comment]
@gasPoweredSticks Web checkout package doesn't support SSR currently! we are planing to support it soon inshallah.

**Comment by Lunchb0ne** [comment]
Any updates on this? We are tying to integrate it in our nextjs app, but even if we use dynamic loading and disable ssr, it keeps throwing window undefined errors when building for production.

**Comment by FaysalMaqbool** [comment]
@halaq  any update yet?

**Comment by MGHZ7** [comment]
Are you still planning ?

**Comment by Ahsan-Ullah1871** [comment]
Is there anyone who can fix this issue?
 or is there an update available @halaq ?
  I am also encountering the same issue in my next.js project.

**Comment by MGHZ7** [comment]
Yes, I encourage you to use the CDN version, It has solved the problem
